### DISCUSSION TITLE
en/docs/Robot_development/quick_start/preparation

### DISCUSSION THREAD

**Discussion by giscus** [D-Robotics/rdk_doc#2] [General]
# en/docs/Robot_development/quick_start/preparation

TogetheROS.Bot supports installation on the RDK with  Ubuntu 20.04/22.04 system. Installing through DEB packages on Ubuntu system is simple and recommended for users who want to experience it initially.

http://localhost:3000/en/docs/Robot_development/quick_start/preparation

**Comment by xuguosheng110** [comment]
great!!!
